lib/timers/promises.js

```diff
@@ -7,13 +7,19 @@ const {
   ReflectConstruct,
   SafePromisePrototypeFinally,
   Symbol,
+  globalThis,
 } = primordials;
 
 const {
   Timeout,
   Immediate,
   insert
 } = require('internal/timers');
+const {
+  clearImmediate,
+  clearInterval,
+  clearTimeout,
+} = globalThis;
```

Comment:
Should these come from timers instead of globalThis?

Reply:
Or would that be cyclical? In that case there should be a comment?

Reply:
It was probably related to a cyclical dependency, however using `require('timers')` doesn't seem to make any test fail, so maybe the cyclical deps was resolved since then? Anyway, pushed a commit, let's see if the CI sees something wrong.